docs/input.dox

```diff
@@ -494,6 +494,20 @@ a mouse button callback.
 glfwSetMouseButtonCallback(window, mouse_button_callback);
 @endcode
 
+@anchor GLFW_DISABLE_MOUSE_BUTTON_LIMIT
+To handle all mouse buttons, instead of only ones with associated
+[button tokens](@ref buttons), set the @ref GLFW_DISABLE_MOUSE_BUTTON_LIMIT
+input mode.
```

Comment:
We need to make clear that this only works for the callback, and not for `glfwGetMouseButton`, with a sentence here, also in `glfw3.h` in the doc comments above `glfwSetInputMode` and `glfwGetMouseButton`.

Reply:
I assumed this would be redundant, as this section of the documentation talks about the callback specifically, and when `glfwGetMouseButton` is mentioned later, it's explicitly said that it only works for named mouse buttons.

If you think clarifying it would be good regardless, I can do that.